src/_pytest/fixtures.py

```diff
@@ -535,7 +546,7 @@ def _addfinalizer(self, finalizer: Callable[[], object], scope) -> None:
             finalizer=finalizer, colitem=colitem
         )
 
-    def applymarker(self, marker) -> None:
+    def applymarker(self, marker: Union[str, MarkDecorator]) -> None:
```

Comment:
Should be plain marks as well

Reply:
Thanks for the review.

It seems that plain mark is not accepted here -- this function calls `self.node.add_marker(marker)` which starts with

```py
        from _pytest.mark import MARK_GEN

        if isinstance(marker, MarkDecorator):
            marker_ = marker
        elif isinstance(marker, str):
            marker_ = getattr(MARK_GEN, marker)
        else:
            raise ValueError("is not a string or pytest.mark.* Marker"
```